packages/block-library/src/button/editor.scss

```diff
@@ -76,6 +80,8 @@
 	}
 }
 
-.wp-block-button-wrapper {
+// Use display table for block previews.
```

Comment:
Conceptually, I don't think we should be having styles that target only previews.
The reason for the display: "table" is that the button container should only wrap the content and not takes the full width. I wonder if there's a way to keep the display table and fix the centering?

Reply:
Hi @youknowriad, thank you for the context I submitted an alternative solution that keeps the display table and fixes the centering.